app/lib/paypal_client.rb

```diff
@@ -0,0 +1,102 @@
+# frozen_string_literal: true
+
+module PaypalClient
+  module_function
+
+  require 'net/http'
+
+  STATUS = {
+    approved:  'APPROVED',
+    active:    'ACTIVE',
+    pending:   'APPROVAL_PENDING',
+    suspended: 'SUSPENDED',
+    expired:   'EXPIRED',
+    cancelled: 'CANCELLED'
+  }.freeze
+
+  PAYPAL_DOMAIN = ENV['PAYPAL_DOMAIN']
+
+  def get_subscription reference_id
+    result = request_get '/v1/billing/subscriptions/' + reference_id
```

Comment:
```suggestion
    result = request_get "/v1/billing/subscriptions/#{reference_id}"
```

No se como no lo _vi_ antes, si es lo único lo hacemos post merge.

Reply:
Dale, lo meto en el del blockeo (después de resolver los posibles 3mil conflictos jaja)